.gitlab-ci.yml

```diff
@@ -1,6 +1,10 @@
 include:
   - project: 'Northern.tech/Mender/mendertesting'
     file: '.gitlab-ci-github-status-updates.yml'
+  - project: 'Northern.tech/Mender/mendertesting'
+    file: '.gitlab-ci-check-commits.yml'
```

Comment:
This repo is not part of the release (and is deprecated, too), but let's use `check-commits-signoffs` instead of `check-commits`.

Reply:
Fixed, thanks!